_posts/en/newsletters/2025-01-10-newsletter.md

```diff
@@ -0,0 +1,175 @@
+---
+title: 'Bitcoin Optech Newsletter #336'
+permalink: /en/newsletters/2025/01/10/
+name: 2025-01-10-newsletter
+slug: 2025-01-10-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Investigating mining pool behavior before fixing a Bitcoin Core bug:**
+  Abubakar Sadiq Ismail [posted][ismail double] to Delving Bitcoin about
+  a [bug][bitcoin core #21950] discovered in 2021 by Antoine Riard that
+  results in it reserving 2,000 vbytes in block templates for coinbase
+  transactions rather than the intended 1,000 vbytes.  Each template
+  could include approximately five additional small transactions if the
+  double reservation was eliminated.  However, that could lead to miners
+  who depended on the double reservation producing invalid blocks,
+  resulting in a large loss of income.  Ismail analyzed past blocks to
```

Comment:
```suggestion
  resulting in a significant loss of income.  Ismail analyzed past blocks to
```

Reply:
The block reward is typically worth $300,000 to $500,000 USD.  Maybe this is just an indicator that my stack isn't big enough, but I still think a half-mil qualifies as large.  :smile: